backend/readers_backend/readers_backend/templates/index.html

```diff
@@ -5,17 +5,218 @@
+                {% endfor %}
+            </div>
+        {% else %}
+            <p class="no-collection-message">No workbooks are currently available. Please check back later.</p>
+        {% endif %}
+    </div>
     {% if latest_us_collection %}
-    <h2>Latest Collection: {{ latest_us_collection.major_version }}.{{ latest_us_collection.minor_version }}</h2>
-    {% for workbook in latest_us_collection.workbooks.all %}
-    <a target="_blank" rel="noopener noreferrer" href="{{workbook.pdf.url}}">
-        <img src="{% static 'covers/' %}{{workbook.number}}.png" alt="My image">
-    </a>
-    {% endfor %}
+    <p class="version-text">Version {{ latest_us_collection.major_version }}.{{ latest_us_collection.minor_version }} {{ latest_us_collection.localization }}</p>
     {% endif %}
 </body>
+
+<style>
+    /* Basic Reset & Global Styles */
+    *,
+    *::before,
+    *::after {
+        box-sizing: border-box;
+        margin: 0;
+        padding: 0;
+    }
+
+    html {
+        font-size: 16px; /* Base font size */
+        scroll-behavior: smooth;
+    }
+
+    body {
+        font-family: 'Inter', 'Roboto', -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji";
+        background-color: #f9fafb; /* Very light gray, almost white */
+        color: #1f2937; /* Darker gray for better contrast */
+        line-height: 1.6;
+        display: flex;
+        flex-direction: column;
+        min-height: 100vh;
+        -webkit-font-smoothing: antialiased;
+        -moz-osx-font-smoothing: grayscale;
+    }
+
+    /* Header */
+    header {
+        background-color: #ffffff;
+        padding: 1.25rem 2rem; /* 20px 32px */
+        box-shadow: 0 1px 3px rgba(0, 0, 0, 0.05), 0 1px 2px rgba(0, 0, 0, 0.03);
+        text-align: center;
+        border-bottom: 1px solid #e5e7eb; /* Subtle border */
+    }
+
+    .logo {
+        max-height: 48px; /* Slightly reduced for a sleeker look */
+        width: auto;
+        vertical-align: middle;
+    }
+
+    /* Main Content Container */
+    .container {
+        max-width: 1280px; /* Common max-width for modern layouts */
+        margin: 0 auto;
+        padding: 2.5rem 2rem; /* 40px 32px */
+        flex-grow: 1; /* Allows container to expand and push footer down */
+        width: 100%;
+    }
+
+    /* Workbook Grid */
+    .workbook-container {
+        display: grid;
+        grid-template-columns: repeat(auto-fill, minmax(240px, 1fr)); /* Slightly larger cards */
+        gap: 2rem; /* 32px */
+    }
+
+    /* Individual Workbook Link/Card */
+    .workbook-link {
+        display: block;
+        position: relative;
+        border-radius: 12px; /* Softer, more modern rounding */
+        overflow: hidden;
+        background-color: #ffffff; /* Card background */
+        box-shadow: 0 4px 6px -1px rgba(0, 0, 0, 0.07), 0 2px 4px -2px rgba(0, 0, 0, 0.04);
+        transition: transform 0.25s ease-in-out, box-shadow 0.25s ease-in-out;
```

Comment:
Shift animations (the transition rules) into a prefers-reduced-motion: no-preference media query body.